src/Database/Schema/FirebirdSchema.php

```diff
@@ -522,7 +522,7 @@ protected function translateSimpleColumnTypes(array &$info)
                 $info['type'] = 'timestamp';
                 $default = (isset($info['default'])) ? $info['default'] : null;
                 if (!isset($default)) {
-                    $default = 'CURRENT_TIMESTAMP';
+                    $default = 'NOW';
```

Comment:
@daniilly let's use `LOCALTIMESTAMP` in order to be compatible with both Firebird 3.0.4 and Firebird 4.